README.md

```diff
@@ -26,33 +26,75 @@ The reason for the export is that one of the language detection packages,
 problems (narrowing conversions) that were once only warnings are now considered
 errors by modern compilers. The export tells them to disregard these errors.
 
-## From start to end
+## Manager webapp
+
+You can use this package either using command line commands or by the manager webapp interface.
+The webapp uses a sqlite DB.
+
+Once you have set up its configuration (see below), you can launch 
+the manager webapp server with uvicorn:
+```
+uvicorn app.main:app
+```
+
+In the manager you can create individual steps or entire pipelines. Steps come with
+default parameters which you can change. The same is true for pipelines. Once the proper
+parameteres have been given to a pipeline, you can "spawn" the steps that constitute it.
+The pipeline will set the parameters of those steps based on its own parameters.
+
+### Configuring the webapp
+
+To use the webapp you must set up a configuration file for it.
+
+First, create it from the template:
+
+```
+cp app/config_example.yaml app/config.yaml
+```
+
+Then open it with an editor and following the instructions fill out the required fields.
+
+### Autorunner
+
+If you set a pipeline's status to "autorun" it flags it for being available to the autorun
+feature. You can start that feature by the autorun button, and it will try to go through
+all the steps of all the "autorun" flagged pipelines.
+
+### Standalone Autorunner
+
+The pipe_that_line.py script takes its command line parameters to create a pipeline,
+spawn its corresponding steps and start the autorunner feature.
+
+It requires the webapp to be configured properly (see above).
+
+## Manually running the scripts
 
 This section describes how to download and process the Common Crawl data to
-arrive at a corpus.
+arrive at a corpus. 
+
+Note: there are several optional command line parameters beyond the ones
+covered in this overview. Use the -h flag to get information about them.
+
+Most scripts support multiprocessing. You can activate it with the -P x
+command line argument, where x is the number of processes.
 
 ### Download the index
 
 The script `get_indexfiles.py` can be used to download the index for a
 specific collection. The example below downloads the index for all pages
-in the `.hu` domain in January 2019:
+in the `elte.hu` domain in the January (week 04) 2019 common crawl batch:
```

Comment:
Was this intentional? Why? To have a "toy" domain for testing?

Reply:
Yes, and also to show the syntax on how to do a non-TLD filtering. So the readers will we don't write *.elte.hu, etc.